### ISSUE TITLE
Error with bootstrap new C# file

### ISSUE THREAD

**Issue by Sofistico** [MoaidHathot/dotnet.nvim#11]
When i try to run the bootstrap command, it returns an error and show this error:

C:/Users/user/source/project/ is not accessible by the current user!
C:/Users/user/project/ is not accessible by the current user!
C:/Users/project/ is not accessible by the current user!
C:/plataforma-biometria-api/ is not accessible by the current user!
plataforma-biometria-api/ is not accessible by the current user!

the absolute path to the root of the project is
C:/Users/user/source/repos/project/project.sln

And then when i type enter or any command for it to continue, of the command, the file name is correct, but the namespace is defined as:
namespace C:;

**Comment by MoaidHathot** [comment]
Hi @Sofistico, thank you for reporting this issue.
It seems there is something strange happening with the permissions the nvim process have in your file system. Basically the extensions is trying to find the path of the project (containing the *.sln file) but since due to the permissions issue, it can't find it and is keeping retracting to higher directories.

Do you have any non-standard permissions model?

**Comment by Sofistico** [comment]
It is a work machine. the administrator of the machine is my company, but my user has full access, when i run this command:
Get-Acl | fl
Which returned this:
Path   : Microsoft.PowerShell.Core\FileSystem::C:\Users\me\source\repos
Owner  : BUILTIN\Admins
Group  : COMPANY\UsersDomain
Access : AUTORIDADE NT\SYSTEM Allow  FullControl
         BUILTIN\Admin Allow  FullControl
         COMPANY\ME Allow  FullControl
Audit  :
Sddl   : O:BAG:DUD:(A;OICIID;FA;;;SY)(A;OICIID;FA;;;BA)(A;OICIID;FA;;;S-1-5-21-2137530444-2073163567-4201480237-11632)

**Comment by Sofistico** [comment]
Is there some command or something that i can run to see and/or some other stuff that can be done?

**Comment by Sofistico** [comment]
Another thing that happens frequently, is when i bootstrap a file, it gets only the last file, like this:

```cs
namespace Animations;

public class KaboomAnimation
{
 
}
```
where the path from nvim is:
~\source\repos\MagiRogue\src\MagusEngine\Components\SpellComponents\Animations\KaboomAnimation.cs

**Comment by MoaidHathot** [comment]
- Do you have any other drive (not C:\) or could you please try it on a path that is not `C:\Users/**` or `C:\**`? For example, can you try with a project (could be a totally new project) on `C:\repo\project`?

- Regarding `KaboomAnimation`, where is the *.csproj file located? If you can provide a skeleton project that reproduces this issue it will be easier for me to debug it. Usually that means the *.csproj was not found.

**Comment by Sofistico** [comment]
I don't have any other drive aside from C:

the sln is on the root of the project, on the MagiRogue folder, here is the project:
https://github.com/Sofistico/MagiRogue/tree/refining-something

Here is an empty file inside the project that when i try to bootstrap, it generates weirdly:
https://github.com/Sofistico/MagiRogue/blob/refining-something/src/MagusEngine/Bus/UiBus/ShowGlyphOnConsole.cs

If it isn't much to ask, can you clone the project on that branch and try to bootstrap the file?

**Comment by MoaidHathot** [comment]
@Sofistico, yes. I cloned it and I think I can reproduce the issue. The bootstrapped namespace is `UiBus` instead of `MagusEngine.UiBus`.
Let me debug it further. Thanks for reporting this!

**Comment by KarpenkoY** [comment]
I have the same issue. Nothing special with rights, but getting

> C:/Users/user/source/project/ is not accessible by the current user! C:/Users/user/project/ is not accessible by the current user! 
C:/Users/project/ is not accessible by the current user! 
C:/plataforma-biometria-api/ is not accessible by the current user! 
plataforma-biometria-api/ is not accessible by the current user!
